runelite-client/src/main/java/net/runelite/client/ui/overlay/components/BackgroundComponent.java

```diff
@@ -38,31 +38,44 @@
 @AllArgsConstructor
 public class BackgroundComponent implements RenderableEntity
 {
+	public static Color DEFAULT_BACKGROUND_COLOR = new Color(70, 61, 50, 156);
 	private static final int BORDER_OFFSET = 2;
-	private static final Color BACKGROUND_COLOR = new Color(70, 61, 50, 156);
-	private static final Color OUTSIDE_STROKE_COLOR = new Color(56, 48, 35, 255);
-	private static final Color INSIDE_STROKE_COLOR = new Color(90, 82, 69, 255);
+
+	@Setter
+	private Color backgroundColor = DEFAULT_BACKGROUND_COLOR;
 
 	@Setter
 	private Rectangle rectangle = new Rectangle();
 
 	@Override
 	public Dimension render(Graphics2D graphics, Point parent)
 	{
+		Color outsideStrokeColor = new Color(
+			Math.max(0, Math.min(255, backgroundColor.getRed() - 14)),
```

Comment:
Doesn't ```getRed() - 14``` always have to be < 255?

Reply:
The min and max methods ensure the value is between 0 and 255.